src/client/user/usersActions.js

```diff
@@ -5,9 +5,9 @@ export const GET_FOLLOWING_COUNT_START = '@users/GET_FOLLOWING_COUNT_START';
 export const GET_FOLLOWING_COUNT_SUCCESS = '@users/GET_FOLLOWING_COUNT_SUCCESS';
 export const GET_FOLLOWING_COUNT_ERROR = '@users/GET_FOLLOWING_COUNT_ERROR';
 
-export const getAccountWithFollowingCount = ({ name }) => dispatch =>
+export const getAccountWithFollowingCount = ({ name, authUser }) => dispatch =>
```

Comment:
We don't need to pass `authUser` as parameter. We can get access to state from there `getState()` method, and then use selector `getAuthenticatedUser()`.

Reply:
I still have to fix this. For the other review comments, i'll get to it when I've resolved this change of using `getState()`